zk.js/src/utxo.ts

```diff
@@ -534,7 +540,7 @@ export class Utxo {
       // console.log("this.assetsCircuit ", this.assetsCircuit);
 
       // console.log("amountHash ", amountHash.toString());
-      // console.log("this.keypair.pubkey ", this.account.pubkey.toString());
+      // console.log("this.keypair.pubkey ", this.publicKey.toString());
       // console.log("this.blinding ", this.blinding.toString());
       // console.log("assetHash ", assetHash.toString());
       // console.log("this.appDataHash ", this.appDataHash.toString());
```

Comment:
can the logs be removed?

Reply:
I need them for debugging every now that's why I have been leaving them.